api/src/main/java/io/opentelemetry/api/common/Labels.java

```diff
@@ -34,31 +58,41 @@ public void forEach(BiConsumer<String, String> consumer) {
         consumer.accept((String) data.get(i), (String) data.get(i + 1));
       }
     }
+
+    static Labels sortAndFilterToLabels(Object... data) {
+      return new AutoValue_Labels_ArrayBackedLabels(
+          sortAndFilter(data, /* filterNullValues= */ false));
+    }
+
+    @Override
+    public LabelsBuilder toBuilder() {
```

Comment:
same request as https://github.com/open-telemetry/opentelemetry-java/pull/2134/files#r530758547, would you mind making `LabelsBuilder` an interface?

Reply:
done. please take a look